### PR TITLE
[AEM-6690] Kicker is clickable in podpromo. Always white when dark theme

### PR DESCRIPTION
**Pull request by pmourelle**
Fix for kicker:
Ticket: https://jira.natgeo.com/browse/AEM-6690

- always white when --inverse is applied.
- cleaned up classes for it using cx()
- if has `href` attr z-index: 10 makes it be clickable in podpromo.

### CODE REVIEW

File: src/modules/promocard/_styles.scss [hunk 1]
```diff
@@ -246,11 +246,12 @@ $mt3_large: 1280px;
     text-decoration: none;
     z-index: zIndex('promocard-bottom');
 
-    &--linked{
+    &[href] {
+      // if kicker is a link set it above big massive link
       z-index: zIndex('promocard-top');
     }
 
-    a:hover {
```
Comment (alexrqs) [inline src/modules/promocard/_styles.scss]: good lord, thank you @pmourelle

File: src/modules/promocard/components/shared/PromoText.js [hunk 2]
```diff
@@ -12,13 +12,13 @@ class PromoText extends Component {
...
     const kickerIsLink = /<a/g; //check if the kicker contains an <a/> tag, this will affect z-index of class applied
 
-    if(props.config.overlay || props.theme === 'dark' || props.noImages){
-      kickerClasses = 'mt3_color--white mt3_promocard-kicker mt3_promocard-kicker--inverse';
-    }else if(props.text.kicker && (props.text.kicker.url || props.text.kicker.label.match(kickerIsLink))) {
-      kickerClasses = 'mt3_color--black mt3_color--black mt3_promocard-kicker mt3_promocard-kicker--linked';
-    }else{
-      kickerClasses = 'mt3_color--black mt3_promocard-kicker';
-    }
+    const isDarkTheme = props.config.overlay || props.theme === 'dark' || props.noImages;
```
Comment (sebasrgl) [inline src/modules/promocard/components/shared/PromoText.js]: :+1: NICE!!!

**Comment by benjachapman** [comment]
hrm i have a tweak in need of tweaking, just a sec

File: src/modules/promocard/components/shared/PromoText.js [hunk 3]
```diff
@@ -12,13 +12,13 @@ class PromoText extends Component {
...
+    kickerClasses = cx(
+      'mt3_promocard-kicker',
+    {
+      'mt3_color--white mt3_promocard-kicker--inverse': isDarkTheme,
+      'mt3_color--black': !isDarkTheme,
+    })
```
Comment (sebasrgl) [inline src/modules/promocard/components/shared/PromoText.js]: :+1: NICE!!!

**Comment by benjachapman** [comment]
Merging it with y'alls blessing 🙏

**Pull request merged**